tests/test_span.py

```diff
@@ -148,7 +148,7 @@ def test_traceback_without_error():
     assert not s.error
     assert not s.get_tag(errors.ERROR_MSG)
     assert not s.get_tag(errors.ERROR_TYPE)
-    assert not s.get_tag(errors.ERROR_STACK)
+    assert s.get_tag(errors.ERROR_STACK)
```

Comment:
I suggest to improve this test adding a:
```python
ok_("something" in s.get_tag(errors.ERROR_STACK))
```
so that we're sure a Python stack is here (I think just a row or something that is constant in the stack is great). Otherwise this test will pass even `s.get_tag(errors.ERROR_STACK)` includes only an int.

What do you think?